tests.sh

```diff
@@ -128,5 +128,34 @@ test_template_local_path() {
   assertErrorCode 0 -p examples/moon-base template ../custom-template.tmpl -o Dockerfile-custom
 }
 
+test_run_repo_say_branch() {
+  result=$(./shuttle -p examples/repo-project-branched run say)
+  if [[ ! "$result" =~ "something clever" ]]; then
+    fail "Expected output to contain 'something minor', but it was:\n$result"
+  fi
+}
+
+test_run_repo_say() {
+  result=$(./shuttle -p examples/repo-project run say)
+  if [[ ! "$result" =~ "something masterly" ]]; then
+    fail "Expected output to contain 'something', but it was:\n$result"
```

Comment:
As above the error description does not match you expectations.

```suggestion
    fail "Expected output to contain 'something masterly', but it was:\n$result"
```